### DISCUSSION TITLE
How can I become a member of MDN localization team?

### DISCUSSION THREAD

**Discussion by scffs** [mdn/mdn-community#319] [Localization] [answered]
Hi guys!
I really enjoy MDN and I want to become a member of the localization team to help someone.
So in [this](https://developer.mozilla.org/en-US/docs/MDN/Community/Roles_teams#contact_the_mdn_team) article I read I need to reach out to mdn-web-docs-community
I'm not sure if I'm writing to the right place, but I think you can direct me to the right place anyway, thanks!

**Comment by schalkneethling** [comment, marked as answer]
Welcome, @scffs! I am going to loop in the @mdn/localization-team-leads to assist here. Thank you for being so interested.

**Reply by scffs** [reply]
Thanks! I'm waiting for your answer

**Reply by Graywolf9** [reply]
Hi! and welcome @scffs it's a pleasure to read about your interest, do you have any prefered lang?

**Reply by scffs** [reply]
> Hi! and welcome @scffs it's a pleasure to read about your interest, do you have any prefered lang? 

Hi, I prefer RU / EN languages

**Reply by Graywolf9** [reply]
Great! I ping to @mdn/yari-content-ru for the help of some one of that lang, also you are free to make all PRs or issues as you want without any special requirement

For `ru` in: https://github.com/mdn/translated-content/
For `en-US` in: https://github.com/mdn/content/

**Reply by scffs** [reply]
Thanks for that
I appreciate it❤

**Reply by sashasushko** [reply]
Hi folks. Unfortunately, I have no rights to edit member's list for RU locale.

But yes, since war in Ukraine started, we have troubles with "free hands". I am the only active member of the team now.

So yes, I don't mind to increase active team members count 🙌

**Reply by scffs** [reply]
> Hi folks. Unfortunately, I have no rights to edit member's list for RU locale.
> 
> But yes, since war in Ukraine started, we have troubles with "free hands". I am the only active member of the team now.
> 
> So yes, I don't mind to increase active team members count 🙌

That sounds a little sad but in any case I will try to make MDN better

**Reply by sashasushko** [reply]
@SphinxKnight what do you think? Can we put on hold inactive members and "issue a call" for new ones? Or how we can resolve this problem? There are no problems with contributing to RU locale, but I'm not enough for all the reviews

**Reply by SphinxKnight** [reply]
@sashasushko hi there :)
In the past, for `fr`, I've opened issues and mentioned @mdn/mdn-community-engagement to remove contributors from the team when they were not active anymore and/or decided to leave (see https://github.com/mdn/translated-content/issues/7913) 
The same goes for adding someone to the team (see https://github.com/mdn/translated-content/issues/4275) 

For small teams (like `ru` or `fr`), I would prefer only active people in them (for governance/security reason). If people come back, it shouldn't be a burden to add them back when needed. In other words, if you don't have any sign of some members, feel free to open an issue to ask them to be removed from the team (explicitly stating the reason why so that they can understand afterwards etc.)

That being written, the consequences is that you can quick be alone facing a lot of PRs.
First: thanks a lot @scffs for your contributions :) I'm also glad you started this discussion here.
Second: I would advise to have a "default" policy for PRs that *only edit pages* in order to tackle with ancient reviews. For example, for `fr`, if noone says anything against the content of the PR itself, I would assume good will and merge the PR after a month if noone had time to review it. Of course, your "default" could be different for `ru`: this should be documented somewhere (https://github.com/mdn/translated-content/tree/main/docs or any other resource for the team)
FWIW, @scffs changes seems simple enough to be merged after a quick "scan" (easier written than done)

Once the team is larger, I would also advise to avoid "self-merged" PRs and wait for proper review (and refer to the "default" strategy after some time if no review happened).

Finally, if you have time/resources for that, feel free to "issue a call" for contributors on some networks, @mdn/mdn-community-engagement may help hopefully 🤞 

@sashasushko please let me know on some PRs if I can be of any help from a "non-russian speaking" point of view.

**Reply by scffs** [reply]
As far as I understand your message if @sashasushko creates an issue, then I can join to the team and review other people's PR?
Am I right?
Thanks for the fast answer

**Reply by SphinxKnight** [reply]
yeah if @mdn/yari-content-ru (meaning @sashasushko at the moment) is OK and create an issue asking for you to be added to the team, you could help review other people's PR for files under `files/ru`.
Joining the team is actually a consequence of the issue where an admin from @mdn/mdn-community-engagement has been notified and adds you (in other words it's not automatic, someone has to manually do something)

**Reply by sashasushko** [reply]
@SphinxKnight thanks for information. I'll do it till end of week

**Comment by scffs** [comment]
By the way some of my PRs have not been reviewed after a month
Maybe there's something wrong with them? At the same time, some new PRs have already been reviewed


![изображение](https://user-images.githubusercontent.com/67464545/212900130-d35b32b7-d355-4f1f-958c-66825f0dde9f.png)

**Reply by sashasushko** [reply]
Because of my resources are limited I'm choosing firstly PR which includes core fixes, then bugs, then everything else

**Reply by scffs** [reply]
Did some people from team leave you alone because of some things in the world or no? Just interesting

**Reply by sashasushko** [reply]
I think they just have more important things right now.

The first months I couldn't find power to contribute to open source too.

So, I hope they're doing well, and some time they will be back 🤞

**Comment by sashasushko** [comment]
I managed to come back half of the team, so I'm expecting that the translation will go faster.

@scffs your PR's will be reviewed soon 🙏

**Reply by scffs** [reply]
it sounds so good
thanks you for your work

**Comment by pransh15** [comment]
Closing this since it was answered.
